src/common/styles/fonts.css

```diff
@@ -0,0 +1,9 @@
+.boldTextOne {
```

Comment:
Common css can be shared accross the entire project. This will help us get the same text sizes/weight etc across the entire site. 

To add additional common classes we can create a name.css file (not name.module.css) Add your class name (make unique) and then you will need to import it into index.css file. 

We should try to keep similar things in common files. 
This is a good example of what a global font would look like.